examples/complete-vpc/main.tf

```diff
@@ -142,3 +146,49 @@ module "vpc" {
     Endpoint = "true"
   }
 }
+
+# Data source used to avoid race condition
+data "aws_vpc_endpoint" "dynamodb" {
```

Comment:
Should there be a filter by type (Interface / Gateway) in this data-source also if this would be for S3 VPC Endpoint?